docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1209,49 +1206,163 @@ visual inspection and quantitatively using an appropriate statistical
-can manually inspect a sample of 10-100 SSNs, confirm that the
-expected number are missing and that the duplication count follows the
-intended distribution.
+For population initialization coverage, we use the ACS PUMS to estimate the
+foreign-born population of the United States,
+and a DHS report estimating the undocumented population in 2018. [DHS_Unauthorized]_
+
+For coverage at immigration, we use the ACS PUMS to estimate the
+foreign-born population who entered in the last year,
+and a demographic modeling analysis estimating the yearly inflow of undocumented immigrants
+in 2017 (the most recent year reported).
+We assume that all undocumented immigrants are foreign-born. [Fazel-Zarandi_2018]_
+
+Simulation strategy
+^^^^^^^^^^^^^^^^^^^
+
+At all times in the simulation, all simulants have either an SSN **or** an ITIN.
+A simulant should never have both.
+
+SSN/ITIN should remain constant for a given simulant for their entire lifespan.
+
+We never switch someone who has an ITIN to have an SSN.
+
+.. note::
+
+  An SSN and an ITIN should not be treated like the same thing.
+  Logic in the taxes observer depends on which one a simulant has.
+
+SSN generation
+''''''''''''''
+
+**SSNs should be unique**.
+The same SSN should not be assigned to two different simulants.
+
+Following the `SSA algorithm <https://www.ssa.gov/kc/SSAFactSheet--IssuingSSNs.pdf>`_,
+the steps to generate an SSN are as follows:
+
+#. Generate a zero-padded integer between 1 and 899 (inclusive) to use as
+   the first three digits.
+   666 is not allowed; this can be never sampled or assigned to another arbitrary
+   number.
+#. Append a dash.
+#. Generate a zero-padded integer between 1 and 99 (inclusive) to use as
+   the next two digits.
+#. Append a dash.
+#. Generate a zero-padded integer between 1 and 9999 (inclusive) to use as
+   the last four digits.
+
+ITIN generation
+'''''''''''''''
+
+**ITINs should be unique**.
+The same ITIN should not be assigned to two different simulants.
+
+ITIN generation is similar to SSN generation, using these steps:
+
+#. Generate a zero-padded integer between **900 and 999** (inclusive) to use as
+   the first three digits.
+#. Append a dash.
+#. Generate a zero-padded integer between **50 and 65, 70 and 88, 90 and 92, or 94 and 99** (inclusive) to use as
+   the next two digits.
+#. Append a dash.
+#. Generate a zero-padded integer between 1 and 9999 (inclusive) to use as
+   the last four digits.
+
+Slides from the IRS on the ITIN format can be found here: https://www.irs.gov/pub/irs-pdf/p4757.pdf
+
+Population initialization
+'''''''''''''''''''''''''
+
+At population initialization, we follow these rules to initialize an SSN or ITIN for every simulant:
+
+#. If the ACS person sampled was born in the United States (according to the :code:`NATIVITY` column),
+   the new simulant is assigned an SSN.
+#. Otherwise, the simulant is assigned an SSN **74.3%** of the time, and an ITIN the remainder of the time.
```

Comment:
Can you add the source for this 74%?

Reply:
The data sources used to calculate both percentages are described in the "Data sources and analysis" section; do you think they need to be moved or referenced here?

Reply:
Hmm I see, its fine then